packages/eslint-plugin/src/rules/strict-boolean-expressions.ts

```diff
@@ -13,6 +13,7 @@ export type Options = [
     allowNullableBoolean?: boolean;
     allowNullableString?: boolean;
     allowNullableNumber?: boolean;
+    allowNullableEnum?: boolean;
```

Comment:
We'll want to also document this on `strict-boolean-expressions.md`. The new option is good!

Reply:
Added the option to the doc!
https://github.com/typescript-eslint/typescript-eslint/pull/6096/commits/2b71e6cb9b454ef937b443841070d717777b3857